html/signup.html

```diff
@@ -45,20 +47,21 @@ <h2 class='log_tit' tabindex="0"><img src="../resources/images/common/logo_tit.p
               <button type="button" class="toggle-password hide-password-icon" data-target="confirmPassword"></button>
               <div id="error-message" class="error-message">비밀번호가 일치하지 않습니다</div>
             </div>
+            <div class="btn_set">
+              <button type="submit" class="btn_login" title="선택됨">로그인</button>
```

Comment:
💊 제안
title 속성은 추가적인 정보를 제공하는 속성이라 지금의 경우는 버튼에 이미 충분한 정보가 있어서 빼는것이 더 적절할 것 같습니다~

```suggestion
              <button type="submit" class="btn_login">로그인</button>
```

[mdn title attr](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)